roles/openshift_metrics/tasks/install_heapster.yaml

```diff
@@ -0,0 +1,13 @@
+---
+- shell: >
+    {{ openshift.common.client_binary }} -n {{ openshift_metrics_project | quote }}
+    --config={{ mktemp.stdout }}/admin.kubeconfig
+    get rc heapster -o jsonpath='{.spec.replicas}' || echo 0
```

Comment:
using `|| echo 0` here seems a bit odd, instead I would set `failed_when: false` here and then do the following below:
```
  replica_count: "{{ heapster_replica_count.stdout | default(0) }}"
```